src/mscorlib/model.xml

```diff
@@ -1051,6 +1051,9 @@
     </Type>
     <Type Name="System.Runtime.Loader.AssemblyLoadContext" condition="FEATURE_HOST_ASSEMBLY_RESOLVER">
       <Member Name="#ctor" />
+      <Member Name="#ctor(System.Boolean)" Condition="FEATURE_COLLECTIBLE_ALC"/>
```

Comment:
New APIs can't be conditioned here either since System.Runtime.Loader type-forwards `AssemblyLoadContext` to System.Private.CoreLib. Also, you shouldn't need anything here that would not be in the contract for System.Runtime.Loader, so all the private/internal stuff can be omitted.